### DISCUSSION TITLE
Flectra HQ 3.0 Helm chart available

### DISCUSSION THREAD

**Discussion by enguerr** [flectra-hq/flectra#40] [Show and tell]
Hello, i wanted to share my project to deploy flectra in kubernetes.
I've wrote the helm chart to deploy flectra 3.0.
You can find the project in the link below
[https://github.com/enguerr/flectrahq-helm](https://github.com/enguerr/flectrahq-helm)

**Comment by parthivgls** [comment]
@enguerr looks like PVC should be Read Write Many instead of Read Write Once for multiple pods to access the same code and filestore folder

**Reply by enguerr** [reply]
You're right. Corrected. Maybe i should move this into values.yaml.

**Comment by parthivgls** [comment]
test as well, specially for session management.
